test/extended/util/framework.go

```diff
@@ -1691,15 +1691,16 @@ type podExecutor struct {
 
 // NewPodExecutor returns an executor capable of running commands in a Pod.
 func NewPodExecutor(oc *CLI, name, image string) (*podExecutor, error) {
-	out, err := oc.Run("run").Args(name, "--labels", "name="+name, "--image", image, "--restart", "Never", "--command", "--", "/bin/bash", "-c", "sleep infinity").Output()
```

Comment:
The `image` argument is now ignored.  It should be removed if it isn't needed.

Reply:
Good point, I missed that. But now when I remove the argument, I see https://github.com/openshift/origin/blob/44f501555d157138abba4a023ad71cf2cd4fd581/test/extended/images/signatures.go#L73 and https://github.com/openshift/origin/blob/8bcc3cd462e3666a730a6fe55778392f204c0fc3/test/extended/security/scc.go#L371 use a custom image. We've broken those tests.

I'm assuming we should go back to supporting a custom image now, to keep things the way they were.